app/uk/gov/hmrc/gform/gform/NewFormController.scala

```diff
@@ -470,14 +474,13 @@ class NewFormController(
     } yield r
   }
 
-  private def exitPageHandler(
-    cache: AuthCacheWithoutForm,
-    formTemplate: FormTemplate,
-    continue: (AuthCacheWithoutForm, FormTemplate) => Future[Result]
+  private def noFormEval(
```

Comment:
Can we name this method more meaningfully? (And not just this one particular method)